fido-2-derivation-nicolas.md

```diff
@@ -41,8 +41,11 @@ Inputs: rpIdHash (32B, e.g. H(example.com)), userId (RP-assigned and password-ve
 Construct:
 
 - version = 1 = 0x01
+- seed_generator_type = byte (0 or byte absent => unspecified, 1 => DiceKeys)
+- seed_generators_state = byte[] (0 to n bytes for n <= some constraint set by authenticator)
```

Comment:
The authenticator can't set the constraints as the construction is meant to be portable among *all* compliant authenticators.

Reply:
Thinking about this again, I'd really like to avoid "configurability", so if possible for the purposes of the "keys" spec, I'd like to treat this "additional metadata" (seed generator related or not) as opaque / independent of the "keys" spec.